### DISCUSSION TITLE
导入obs后显示403

### DISCUSSION THREAD

**Discussion by ScarlettLsx** [xfgryujk/blivechat#229] [提问]
如图
![Uploading 7141c4a4-728e-45e2-9338-9d66210f1925.png…]()

**Comment by ScarlettLsx** [comment]
![7141c4a4-728e-45e2-9338-9d66210f1925](https://github.com/user-attachments/assets/212bdb95-746a-4fa8-920d-961e441d07e2)

**Comment by xfgryujk** [comment]
更新到v1.9.2然后确保加载器URL改成和新版一样的配置

https://github.com/xfgryujk/blivechat/blob/75938141f0d83c8c2682b658b3fa257c755a7b64/data/config.example.ini#L22
